--- Canna37p3/Canna.conf~	2006-04-13 14:18:56.625048078 +0300
+++ Canna37p3/Canna.conf	2006-04-13 14:25:47.404240386 +0300
@@ -109,12 +109,12 @@
 # define DefErrDir $(cannaPrefix)/log
 # define DefLibCannaDir $(cannaExecPrefix)/lib
 #else 
-# define DefCannaPrefix /usr/local/canna
+# define DefCannaPrefix /usr
 # ifdef ObsoleteInstallDir
 #  define DefErrDir /usr/spool/canna
 #  define DefLibCannaDir /usr/lib
 # else
-#  define DefErrDir /var/log
+#  define DefErrDir /var/log/canna
 #  define DefLibCannaDir $(cannaExecPrefix)/lib
 # endif
 #endif
@@ -169,13 +169,13 @@
 #define ModernElfLink NO /* experimental */
 
 
-cannaOwner = bin
-cannaGroup = bin
+cannaOwner = canna
+cannaGroup = canna
 
 #ifdef InstallAsUser
 cannaOwnerGroup =
 #else
-cannaOwnerGroup = -o $(cannaOwner) -g $(cannaGroup)
+cannaOwnerGroup =
 #endif
 
 cannaDsoRev = 1.2
@@ -199,9 +199,9 @@
 CHGRP = :
 CHMOD = :
 #else
-CHOWN = chown
-CHGRP = chgrp
-CHMOD = chmod
+CHOWN = :
+CHGRP = :
+CHMOD = :
 #endif
 
 /* 日本語マニュアルを使わないのであればコメントアウト */
--- Canna37p3/server/Imakefile~	2006-04-13 14:33:32.394645446 +0300
+++ Canna37p3/server/Imakefile	2006-04-13 14:34:10.555499249 +0300
@@ -6,7 +6,7 @@
 #ifdef __EMX__
          INSTFLAGS = $(INSTUGIDFLAGS)
 #else
-         INSTFLAGS = -s $(INSTUGIDFLAGS)
+         INSTFLAGS = $(INSTUGIDFLAGS)
 #endif
            DEFINES = $(CANNASERVER_DEFINES) $(SIGNAL_DEFINES) ConnectionFlags
           INCLUDES = -I$(CANNAROOT)/include
