--- Canna35b2/ccustom/canna.h.glibc	Wed Nov  6 20:21:45 1996
+++ Canna35b2/ccustom/canna.h	Mon Jun 12 15:13:32 2000
@@ -40,7 +40,7 @@
 
 /*#include "jrkanji.h"*/
 
-#if defined(SYSV) || defined(SVR4) || defined(__STDC__)
+#if defined(SYSV) || defined(SVR4) || defined(__STDC__) && !defined(linux)
 # if defined(SYSV) || defined(SVR4)
 #  include <memory.h>
 # endif
--- Canna35b2/lib/RK/RKintern.h.glibc	Mon Jun 12 15:18:33 2000
+++ Canna35b2/lib/RK/RKintern.h	Mon Jun 12 15:20:36 2000
@@ -97,7 +97,7 @@
 #include	<ctype.h>
 #endif
 
-#if defined(SYSV) || defined(SVR4) || defined(__STDC__) || defined(WIN)
+#if (defined(SYSV) || defined(SVR4) || defined(__STDC__) || defined(WIN)) && !defined(linux)
 # if defined(SYSV) || defined(SVR4)
 #  include <memory.h>
 # endif
--- Canna35b2/lib/RK/ngram.c.glibc	Mon Jun 12 15:19:21 2000
+++ Canna35b2/lib/RK/ngram.c	Mon Jun 12 15:19:35 2000
@@ -42,7 +42,7 @@
 #include <sys/stat.h>
 #endif
 
-#if defined(SYSV) || defined(SVR4) || defined(__STDC__)
+#if defined(SYSV) || defined(SVR4) || defined(__STDC__) && !defined(linux)
 # if defined(SYSV) || defined(SVR4)
 #  include <memory.h>
 # endif
--- Canna35b2/lib/RKC/rkc.h.glibc	Mon Jun 12 15:17:43 2000
+++ Canna35b2/lib/RKC/rkc.h	Mon Jun 12 15:21:14 2000
@@ -41,7 +41,7 @@
 #include "RKrename.h"
 #endif
 
-#if defined(SYSV) || defined(SVR4) || defined(__STDC__) || defined(WIN)
+#if (defined(SYSV) || defined(SVR4) || defined(__STDC__) || defined(WIN)) && !defined(linux)
 # if defined(SYSV) || defined(SVR4) || defined(WIN)
 #  include <memory.h>
 # endif
--- Canna35b2/lib/canna/canna.h.glibc	Mon Jun 12 15:14:09 2000
+++ Canna35b2/lib/canna/canna.h	Mon Jun 12 15:14:26 2000
@@ -105,7 +105,7 @@
 
 #define STROKE_LIMIT 500 /* ストロークで接続を切る */
 
-#if defined(SYSV) || defined(SVR4) || defined(__STDC__) || defined(WIN)
+#if defined(SYSV) || defined(SVR4) || defined(__STDC__) || defined(WIN) && !defined(linux)
 # if defined(SYSV) || defined(SVR4)
 #  include <memory.h>
 # endif
--- Canna35b2/server/IR.h.glibc	Mon Jun 12 15:23:47 2000
+++ Canna35b2/server/IR.h	Mon Jun 12 15:24:00 2000
@@ -27,7 +27,7 @@
 #define EXTENSION
 #define DEBUG
 
-#if defined(SYSV) || defined(SVR4) || defined(__STDC__)
+#if (defined(SYSV) || defined(SVR4) || defined(__STDC__)) && !defined (linux)
 # if defined(SYSV) || defined(SVR4)
 #  include    <memory.h>
 # endif
