diff -Nur Canna35b2.orig/Canna.conf Canna35b2/Canna.conf
--- Canna35b2.orig/Canna.conf	Thu Nov 28 11:27:01 1996
+++ Canna35b2/Canna.conf	Sat Mar  9 19:13:20 2002
@@ -128,27 +128,27 @@
  * $Id$
  */
 
-cannaBinDir = /usr/local/canna/bin
-cannaSrvDir = /usr/local/canna/bin
-cannaLibDir = /usr/local/canna/lib
-cannaManDir = /usr/local/canna/man
-cannaIncDir = /usr/local/canna/include/canna
+cannaBinDir = /usr/bin
+cannaSrvDir = /usr/sbin
+cannaLibDir = /var/lib/canna
+cannaManDir = /usr/share/man/ja
+cannaIncDir = /usr/include/canna
 libCannaDir = /usr/lib
 
-ErrDir  = /usr/spool/canna
+ErrDir  = /var/log/canna
 
 wcharDefinition = -DCANNA_WCHAR
 Wlib =
-JapaneseLocale = japanese
+JapaneseLocale = ja_JP.eucJP
 
-cannaOwner = bin
-cannaGroup = bin
+cannaOwner = canna
+cannaGroup = canna
 
 cannaOwnerGroup = -o $(cannaOwner) -g $(cannaGroup)
 
 pointerIntegerDef =
 
-cannaDsoRev = 1
+cannaDsoRev = 1.0.0
     /* SunOS 4.x をお使いの方で shared object を利用する場合には
             cannaDsoRev = 1.1
     と指定してください。また、FreeBSD を利用する場合は、
@@ -170,7 +170,7 @@
 #define JAPANESEMAN
 
 /* locale ディレクトリを持っている場合は以下を記述する */
-/* #define LOCALE_DIR /usr/lib/locale */
+#define LOCALE_DIR /usr/share/locale
 
 /* エンジンスイッチ機能を使うのであればコメントをはずす */
 /* ※注 Canna version 3.5 ではコメントをはずさないでください */
